test/soak/soak_test.go

```diff
@@ -9,11 +9,18 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/aws/amazon-cloudwatch-agent-test/environment"
```

Comment:
Does soak_test run as part of the same GHA as others or a diff action? Maybe drop the flag being passed in there since it doesnt look like its being used for this test?

Reply:
It does. It simplifies github workflows to rely on the integration test terraform code.